src/pages/waiting-room-to-car/cat-c/waiting-room-to-car.cat-c.page.ts

```diff
@@ -0,0 +1,319 @@
+
+// TODO: MES-4254 Import cat c selector
+import {
+  hasEyesightTestGotSeriousFault,
+} from '../../../modules/tests/test-data/cat-be/test-data.cat-be.selector';
+
+// TODO: MES-4254 Import cat c reducer
+import { getTestData } from '../../../modules/tests/test-data/cat-be/test-data.cat-be.reducer';
+import { PersistTests } from '../../../modules/tests/tests.actions';
+import { CAT_C } from '../../page-names.constants';
+import { BasePageComponent } from '../../../shared/classes/base-page';
+import { VehicleChecksQuestion } from '../../../providers/question/vehicle-checks-question.model';
+import { TestCategory } from '@dvsa/mes-test-schema/categories/common/test-category';
+import { VehicleChecksScore } from '../../../shared/models/vehicle-checks-score.model';
+
+// TODO: MES-4254 Import cat c selector
+import {
+  getVehicleChecksCatBE,
+} from '../../../modules/tests/test-data/cat-be/vehicle-checks/vehicle-checks.cat-be.selector';
+import { FaultCountProvider } from '../../../providers/fault-count/fault-count';
+
+// TODO: MES-4254 Import cat c schema
+import { CatBEUniqueTypes } from '@dvsa/mes-test-schema/categories/BE';
+import { VehicleChecksCatCComponent } from './components/vehicle-checks/vehicle-checks.cat-c';
+
+interface WaitingRoomToCarPageState {
+  candidateName$: Observable<string>;
+  registrationNumber$: Observable<string>;
+  transmission$: Observable<GearboxCategory>;
+  schoolCar$: Observable<boolean>;
+  dualControls$: Observable<boolean>;
+  instructorAccompaniment$: Observable<boolean>;
+  supervisorAccompaniment$: Observable<boolean>;
+  otherAccompaniment$: Observable<boolean>;
+  interpreterAccompaniment$: Observable<boolean>;
+  eyesightTestComplete$: Observable<boolean>;
+  eyesightTestFailed$: Observable<boolean>;
+  gearboxAutomaticRadioChecked$: Observable<boolean>;
+  gearboxManualRadioChecked$: Observable<boolean>;
+  vehicleChecksScore$: Observable<VehicleChecksScore>;
+
+  // TODO: MES-4254 Use cat c type
+  vehicleChecks$: Observable<CatBEUniqueTypes.VehicleChecks>;
+}
+
+@IonicPage()
+@Component({
+  selector: '.waiting-room-to-car-cat-c-page',
+  templateUrl: 'waiting-room-to-car.cat-c.page.html',
+})
+export class WaitingRoomToCarCatCPage extends BasePageComponent {
+  pageState: WaitingRoomToCarPageState;
+  form: FormGroup;
+
+  @ViewChild(VehicleChecksCatCComponent)
+  vehicleChecks: VehicleChecksCatCComponent;
+
+  showEyesightFailureConfirmation: boolean = false;
+
+  tellMeQuestions: VehicleChecksQuestion[];
+
+  constructor(
+    public store$: Store<StoreModel>,
+    public navController: NavController,
+    public navParams: NavParams,
+    public platform: Platform,
+    public authenticationProvider: AuthenticationProvider,
+    public faultCountProvider: FaultCountProvider,
+    public questionProvider: QuestionProvider,
+  ) {
+    super(platform, navController, authenticationProvider);
+    this.tellMeQuestions = questionProvider.getTellMeQuestions(TestCategory.BE);
```

Comment:
I assume ```this.tellMeQuestions = questionProvider.getTellMeQuestions(TestCategory.BE);``` can't be change until the schema etc is implemented? Just highlighting. :)

Reply:
Yeah that's right